.github/workflows/sync-issue-with-pr.yaml

```
@@ -0,0 +1,62 @@
+          # Get the PR body
+          PR_BODY=$(gh pr view ${{ github.event.pull_request.number }} --json body --jq '.body')
+
+          # Use regex to find issue numbers referenced by "Close"
+          ISSUE_NUMBERS=$(echo "$PR_BODY" | grep -oP '(?<=Close #)[0-9]+')
+
+          echo "Linked issues: $ISSUE_NUMBERS"
+          echo "::set-output name=issues::$ISSUE_NUMBERS"
+
+      - name: Read Issue Labels and Titles
+        id: issue_data
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2043:warning:3:21: This loop will only ever run once. Bad quoting or missing glob/expansion?

(shellcheck)